File: dukevineland.client/src/app/import-export/models/export-options-response-dto.ts
export interface ExportOptionsResponseDto {
  documentTypes: IDocumentType[];
  exportOptions: IExportOption[];
}

export interface IDocumentType {
  name: string;
  description: string;
}

export interface IExportOption {
  name: string;
  description: string;
  supportedDocumentTypes: string[];
}

File: DukeVineland.Dtos/ImportExportDtos/ExportOptionsResponseDto.cs
namespace DukeVineland.Dtos.ImportExportDtos
{
    public class ExportOptionsResponseDto
    {
        public DocumentType[] DocumentTypes { get; set; } = Array.Empty<DocumentType>();
        public ExportOption[] ExportOptions { get; set; } = Array.Empty<ExportOption>();
    }

    public class DocumentType
    {
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
    }

    public class ExportOption
    {
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public string[] SupportedDocumentTypes { get; set; } = Array.Empty<string>();
    }
}
